R/report.R

```diff
@@ -184,10 +184,8 @@ num_deps <- function(pkg) {
 }
 
 pkg_source_link <- function(chk) {
-  if (chk$cran) {
+  if (!is.null(chk$cran)) {
```

Comment:
I think you also need to check of it is `TRUE`, so `isTRUE()` might be the best here.

Reply:
I just tried that and it produced fewer good links than before

Reply:
Interesting.